pandas/tests/indexes/timedeltas/test_timedelta.py

```diff
@@ -509,6 +509,10 @@ def test_fields(self):
         tm.assert_series_equal(s.dt.seconds, Series(
             [10 * 3600 + 11 * 60 + 12, np.nan], index=[0, 1]))
 
+        # preserve name (GH15589)
```

Comment:
that would indeed be a possibility, and just checked and eg also `freq` is included in this list (which has a different return type). So would start skipping those, which would also not be that clean.